src/request-pipeline/header-transforms/index.js

```diff
@@ -0,0 +1,59 @@
+import {
+    requestTransforms,
+    forcedRequestTransforms,
+    responseTransforms,
+    forcedResponseTransforms
+} from './transforms';
+
+// Transformation routine
+function transformHeaders (srcHeaders, ctx, transformList, forcedTransforms) {
```

Comment:
@miherlosev 
> * Move `transformHeaders` function to the `src/request-pipeline/header-transforms/index.js` file.
> * Rename function parameters as follows: `(headers, ctx, transforms, forcedTransforms)`


I renamed `forced` to `forcedTransforms`. But if i rename `srcHeaders` to `headers` and `transformList` to `transforms` i'll get lint errors: `'...' is already declared in the upper scope no-shadow` because of the `const applyTransforms = function (...)` of line 12.

I left `srcHeaders` and `transformList` parameters just like it was, but what is the preferable approach?

Reply:
OK, let it as is. This code part requires more complex refactoring. We will do it later.